src/domain/runtime/type/remote_object/type/object/property_preview.rs

```diff
@@ -71,6 +71,18 @@ impl PropertyPreview {
                         value.to_string()
                     }
                 }
+                Type::Accessor => {
+                    let r#type = if let Some(subtype) = &self.subtype {
+                        format!("ttt{:?}", subtype)
```

Comment:
what is ttt?

Reply:
Oh shoot that was for testing I need to remove that